src/storage/common_storage.ts

```diff
@@ -572,19 +576,26 @@ export function makeModuleContent(
   pythonCode: string,
   blocksContent: string,
   methodsContent: string,
-  componentsContent: string): string {
+  componentsContent: string,
+  optVersionContent?: string): string {
```

Comment:
Not sure we want this optional.   When one is uploaded without it, we are giving it a default value.